### ISSUE TITLE
Bug: $updateCommands

### ISSUE THREAD

**Issue by dogukancim** [aoijs/aoi.js#602]
### Bug Type

Function

### Bugged Function

```js
module.exports = [
    {
        name: "rr",
        $if: "old",
        code: `
        $if[$authorID==$clientOwnerIDs]
            $updateCommands
            Hot reload successfully completed.
        $endif
        `
    }
];
```

### Version of aoi.js

v6.7.1

### Version of Node.js

v20.10.0 (LTS)

### Excepted Output

Files with the old loader _`loader.load(bla bla bla, "./commands")`_ can be reloaded with $updateCommands, but this does not work on the new loader _`client.loadCommands("path", log true/false)`_.

![image](https://github.com/aoijs/aoi.js/assets/78941156/cd77f217-3f3e-47ee-8c9e-fd769480dea2)
![image](https://github.com/aoijs/aoi.js/assets/78941156/a9732105-5c8b-44ab-a985-a69e45db935a)

### Relevant log output

![image](https://github.com/aoijs/aoi.js/assets/78941156/6f167d03-a87b-4263-91fc-2de7d12cfb4e)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Faf4a** [comment]
can't reproduce..?

**Comment by supremesupreme** [comment]
same lol

**Comment by valentineu0** [comment]
The situation is still the same
![image](https://github.com/aoijs/aoi.js/assets/78941156/944ee4fd-8df9-4226-a0ca-7693101b0dd0)
![image](https://github.com/aoijs/aoi.js/assets/78941156/71992443-3ed5-4b0d-8737-f0130697fdaf)

**Comment by valentineu0** [comment]
Currently the command closes directly.
![image](https://github.com/aoijs/aoi.js/assets/78941156/71fd2786-613a-4e39-842d-2ab33f47b30c)

**Comment by dogukancim** [comment]
Still command not updating unless bot is restarted

**Comment by USERSATOSHI** [comment]
does this still happen with v6.8.0?

**Comment by dogukancim** [comment]
After v6.8, a single loadCommands runs the code properly, but when two different folders are requested to be loaded, a single loadCommands takes effect.

![image](https://github.com/aoijs/aoi.js/assets/78941156/d1469d20-4227-48c7-b7cf-cfd67d09a25d)
![image](https://github.com/aoijs/aoi.js/assets/78941156/92db49c6-4c53-46b1-a811-d797d037d2ed)
![image](https://github.com/aoijs/aoi.js/assets/78941156/b29ecea1-3dc6-4d9d-a321-af7e701eaebd)

The reason why the error report was opened was fixed in v6.8 *(except for the part I additionally stated above)*.

**Comment by USERSATOSHI** [comment]
> After v6.8, a single loadCommands runs the code properly, but when two different folders are requested to be loaded, a single loadCommands takes effect.
> 
> ![image](https://github.com/aoijs/aoi.js/assets/78941156/d1469d20-4227-48c7-b7cf-cfd67d09a25d)
> ![image](https://github.com/aoijs/aoi.js/assets/78941156/92db49c6-4c53-46b1-a811-d797d037d2ed)
> ![image](https://github.com/aoijs/aoi.js/assets/78941156/b29ecea1-3dc6-4d9d-a321-af7e701eaebd)
> 
> The reason why the error report was opened was fixed in v6.8 *(except for the part I additionally stated above)*.
> 
> 

this is intended as client.loadCommands doesn't do a single loader , 

it creates a loader , loads the cmd and closes itself

which is why if u use the second one , the first one would get override 

I recommend using the old setup if u are doing multiple folder loading 👍

**Comment by Leref** [comment]
Issue been solved above, if there is needed for nested it can be added quickly but doesn't seem necessary.
